### PR TITLE
Added update staking rewards params

### PR DESCRIPTION
**Pull request by Jedi2002**

### CODE REVIEW

File: mint-params.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "mint_denom": "rowan",
+  "inflation_rate_change": "0.13",
+  "inflation_max": "0.67",
+  "inflation_min": "0.47",
+  "goal_bonded": "0.78",
+  "blocks_per_year": 650000
+}
+
```
Comment (joneskm) [inline mint-params.json:9]: Should this file be committed?

File: minter.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+  "inflation": "0.42",
+  "annual_provisions": "500000000000000.000000000"
+}
+
```
Comment (joneskm) [inline minter.json:5]: Should this file be committed?

File: proto/sifnode/clp/v1/tx.proto [hunk 3]
```diff
@@ -20,8 +20,44 @@ service Msg {
...
+//    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec",
+//    (gogoproto.nullable) = false
+//  ];
+//  string inflation_min = 4 [
+//    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec",
+//    (gogoproto.nullable) = false
+//  ];
+//  string annual_provisions = 5 [
+//    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec",
+//    (gogoproto.nullable) = false
```
Comment (joneskm) [inline proto/sifnode/clp/v1/tx.proto:42]: Should be removed?

File: x/clp/client/cli/tx.go [hunk 4]
```diff
@@ -487,3 +549,22 @@ func GetCmdUnlockLiquidity() *cobra.Command {
 
 	return cmd
 }
+
+func ParseFile(f string, rewardPeriods interface{}) interface{} {
+	//var rewardPeriods []*types.RewardPeriod
+	//signer := clientCtx.GetFromAddress()
+	//filePath := viper.GetString(FlagRewardPeriods)
```
Comment (joneskm) [inline x/clp/client/cli/tx.go:556]: Remove comments

File: x/clp/keeper/msg_server.go [hunk 5]
```diff
@@ -22,6 +22,24 @@ type msgServer struct {
...
+	if err != nil {
+		return nil, err
+	}
+	if !k.tokenRegistryKeeper.IsAdminAccount(ctx, signer) {
+		return nil, errors.Wrap(types.ErrNotEnoughPermissions, fmt.Sprintf("Sending Account : %s", msg.Signer))
+	}
+	if !(msg.Minter.AnnualProvisions.IsZero() && msg.Minter.Inflation.IsZero()) {
+		k.mintKeeper.SetMinter(ctx, msg.Minter)
+	}
+	k.mintKeeper.SetParams(ctx, msg.Params)
```
Comment (joneskm) [inline x/clp/keeper/msg_server.go:37]: Does code for updating the staking rewards belong inside the CLP module?

**Review by canercandan — APPROVED** [review]
@Jedi2002 apart from the minor changes Kevin mentioned this PR LGTM

**Pull request merged**